### PR TITLE
Adds expiration of award

### PR DESCRIPTION
**Pull request by hlapp**
Introduces expiration of the award within 3 months, as discussed.

### CODE REVIEW

**Comment by nlharris** [comment]
+1

**Comment by nlharris** [comment]
@gedankenstuecke what do I click to approve the changes like you did?

File: Travel_fellowships.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ participation in OBF events and member projects.
 ## What does the fellowship cover?
 The fellowship can be used to cover direct travel costs (such as airfare, train, metro, bus and taxi), hotel, child-care and / or conference registration fees up to a value of USD 1,000. Note that rental car expenses and personal car mileage are not eligible for reimbursement. We anticipate that these awards will supplement travel costs, not cover all expenses. If you anticipate that you would require more than USD 1,000 in order to attend the event, please include that information on the application form.
 
-The OBF will reimburse the recipient after the event based on receipts provided and proof of attendance. There is no cash value beyond eligible and documented expenses.
+The OBF will reimburse the recipient after the event based on receipts provided and proof of attendance. There is no cash value beyond eligible and documented expenses. All reimbursement claims must be filed (and all preequisites for receiving payment must be met, see below) within 3 months after conclusion of the event, after which the fellowship award expires.
```
Comment (yochannah) [inline Travel_fellowships.md]: tiny typo - preequisites  should be prerequisites .

Comment (nlharris) [reply Travel_fellowships.md]: well spotted

Comment (hlapp) [reply Travel_fellowships.md]: Thanks, fixed!

**Comment by peterjc** [comment]
Are we going to add something like "Extensions to this deadline are at the discretion of the board if the recipient has mitigating circumstances."?

**Comment by gedankenstuecke** [comment]
I think on the call the consensus was that we'll not advertise this but might handle it like that in practice?

**Comment by hlapp** [comment]
> I think on the call the consensus was that we'll not advertise this but might handle it like that in practice?

Right. The policy does not include a statement that would categorically preclude the Board from granting exceptions.

**Comment by hlapp** [comment]
Looks like we have majority.

**Pull request merged**